README.md

```diff
@@ -19,8 +19,9 @@ Easily implement WebAuthn in your ruby web server
 ### User Agent compatibility
 
 So far, the only browser that have web authentication support are:
-  - Mozilla Firefox Quantum 60+ (Enabled by default).
-  - Google Chrome 65+ (Disabled by default, go to chrome://flags to enable Web Authentication API feature). Note: it is enabled by default in 67+ as stated [here](https://www.chromestatus.com/feature/5669923372138496).
+  - Mozilla Firefox Quantum 60+ ([Enabled by default](https://www.mozilla.org/en-US/firefox/60.0/releasenotes/))
+  - Google Chrome 67+ ([Enabled by default](https://www.chromestatus.com/feature/5669923372138496))
+  - Google Chrome 65+ (Disabled by default, go to chrome://flags to enable Web Authentication API feature)
```

Comment:
What about changing `65+` to `65 & 66`, to avoid any potential confusion?

Reply:
You are right! I'll update. Thanks for catching that.